ci/run_envoy_docker.sh

```diff
@@ -9,13 +9,21 @@ function is_windows() {
   [[ "$(uname -s)" == *NT* ]]
 }
 
+function export_env() {
+    default="$1"
+    alt="$2"
+
+    if [ -z "${!default}" ]; then
+        export "$default"="$alt"
+    fi
+}
+
 read -ra ENVOY_DOCKER_OPTIONS <<< "${ENVOY_DOCKER_OPTIONS:-}"
 
-# TODO(phlax): uppercase these env vars
-export HTTP_PROXY="${http_proxy:-}"
-export HTTPS_PROXY="${https_proxy:-}"
-export NO_PROXY="${no_proxy:-}"
-export GOPROXY="${go_proxy:-}"
+export_env HTTP_PROXY "${http_proxy:-}"
```

Comment:
```suggestion
export HTTP_PROXY "${http_proxy:-${HTTP_PROXY:-}}"
```

Reply:
@phlax 
Thanks for your suggestion! Adjusted the change in the new way.